LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Database/DataBrokerProtectionDataManager.swift

```diff
@@ -19,17 +19,34 @@
 import Foundation
 import Common
 
+public extension Notification.Name {
+    /// Notification posted when a profile is saved.
+    static let pirProfileSaved = Notification.Name("pirProfileSaved")
```

Comment:
Note: I intentional choose this notification name so it’s NOT tied to Freemium. I did this as it could possible be re-used outside of Freemium. This matches the intentions for the `DBPProfileSavedNotifier` protocol.

This naming is in contract to the notification names defined in `FreemiumPIRScanResultPolling`, which are specific to Freemium.